groceries_app/models.py

```diff
@@ -50,6 +50,7 @@ class IngredientQuantity(models.Model):
     quantity = models.FloatField()
     measurement = models.ForeignKey(Measurement, on_delete=models.PROTECT)
     meal_name = models.ForeignKey(Meal, on_delete=models.PROTECT)
+    optional = models.BooleanField(default=False)
```

Comment:
Can you make it more descriptive? `optional_ingredient`?

Reply:
Good idea. Will do.